### ISSUE TITLE
Make `Open Editors` and Project directory resizable panes

### ISSUE THREAD

**Issue by sehyod** [refstudio/refstudio#255]
Currently the list of open editors and the file explorer are two components next to each other. This means that opening a file, make the former longer and moves shifts everything down, which is annoying when trying to work on the file explorer (trying to open the file then right-clicking on it for example)
We should make these components resizable panes, so that the number of open editors don't change the layout.

https://github.com/refstudio/refstudio/assets/58954208/46434bf0-2232-4188-9933-9172650aba15

**Comment by hammer** [comment]
I'm not sure we need the open files listing at all. It's not present in VS Code, for example. Should we just eliminate it?

**Comment by sehyod** [comment]
You can have it in VSCode, but I have personally never used this feature. I don't mind eliminating it from RefStudio. What do you think @cguedes?
<img width="370" alt="image" src="https://github.com/refstudio/refstudio/assets/58954208/766cac3f-7cd7-48d7-9703-3d2ec6b30517">

**Comment by sehyod** [comment]
That's a fair point! But I think we can move these actions to a context menu when right-clicking on tabs, and also add some keyboard shortcuts for them

**Comment by cguedes** [comment]
Agree. That's a solution. We can also have these actions in the command palette (#224)

**Comment by hammer** [comment]
Yes I would prefer for those actions to be in the editor window or command palette or both rather than in the explorer. I have a list of explorer nits that we can discuss Thursday! As a UX philosophy I'd like to limit the actions taken in the left sidebar and use it more as a quick look for informational purposes, with heavy work happening in the editor area.
